lib/src/server/server.dart

```diff
@@ -190,3 +160,49 @@ class Server {
     _secureServer = null;
   }
 }
+
+/// A gRPC server that serves via provided [ServerTransportConnection]s.
```

Comment:
I  find the ordering a bit confusing - I would expect base class to come ahead of child in the file